### PR TITLE
guide(alwaysObtain): explanation for alwaysObtain

### PR DESCRIPTION
**Pull request by S222em**
**Please describe the changes this PR makes and why it should be merged:**
This PR adds an explanation for `alwaysObtain`.

**Status and versioning classification:**
- This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by xHyroM** [comment]
Can you add this to https://gcommands.js.org/guide/guide/commands/additionalfeatures/ ?

File: guide/guide/questions/alwaysobtain.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# What does the alwaysObtain option in commands do?
+
+:::tip
+`alwaysObtain` only affects message commands.
+:::
+
+The `alwaysObtain` option does a simple thing. Arguments in the first message iniating the command do not count.
+
+This example is too showcase in which cases `alwaysObtain` would be usefull:
```
Comment (Caieon) [inline guide/guide/questions/alwaysobtain.md:9]:
```suggestion
This example is to showcase in which cases `alwaysObtain` would be useful:
```

File: guide/guide/questions/alwaysobtain.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# What does the alwaysObtain option in commands do?
+
+:::tip
+`alwaysObtain` only affects message commands.
+:::
+
+The `alwaysObtain` option does a simple thing. Arguments in the first message iniating the command do not count.
```
Comment (Caieon) [inline guide/guide/questions/alwaysobtain.md:7]:
```suggestion
The `alwaysObtain` option does a simple thing. Arguments in the first message initiating the command do not count.
```

**Review by Caieon — CHANGES_REQUESTED** [review]

**Comment by S222em** [comment]
> Can you add this to https://gcommands.js.org/guide/guide/commands/additionalfeatures/ ?

@xHyroM done

**Pull request merged**